_content/tour/por/algorithms-strings.article

```diff
@@ -1,67 +1,66 @@
-String Operations
-This section provides examples that perform string operations.
+Operações com Strings
+Esta seção fornece exemplos que realizam operações em strings.
 
-* Permutation
+* Permutação
 
-- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Watch The Video]]
-- Need Financial Assistance, Use Our [[https://www.ardanlabs.com/scholarship/][Scholarship Form]]
+- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Assista ao Vídeo]]
+- Precisa de Assistência Financeira? Use Nosso [[https://www.ardanlabs.com/scholarship/][Formulário de Bolsa de estudos]]
 
-The sample program implements a check to see if a string is a
-permutation or not.
+O programa exemplo implementa uma verificação para ver se uma string é ou não uma permutação.
 
-- See more at [[https://en.wikipedia.org/wiki/Permutation][https://en.wikipedia.org/wiki/Permutation]]
 
-*Diagram*
+- Veja mais em [[https://pt.wikipedia.org/wiki/Permutação][https://pt.wikipedia.org/wiki/Permutação]]
```

Comment:
I don't think we should change the link.

Reply:
What are you guys doing? @sergiolpf @sergiolpf @rschio

Reply:
I am not translating links. Looking at other languages translations they also did not translate the links.